### PR TITLE
dlb: Add readme file and dlb plugin daemonset yaml, and Fix demo yaml

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
Commit 1:
README.md file and libdlb-demo pod yaml file to have a guaranteed pod

Commit 2:
While writing README.md, I was thinking that it would be stupid to put only instruction without having a real working daemonset yaml. So, I add this commit also in this pr.!

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]
/lgtm

File: cmd/dlb_plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+
+## Installation
+
+The following sections detail how to obtain, build, deploy and test the DLB device plugin.
+
+Examples are provided showing how to deploy the plugin either using a DaemonSet or by hand on a per-node basis.
+
+### Deploy with pre-built container image
+*This part will be updated later if necessary
```
Comment (mythi) [inline cmd/dlb_plugin/README.md:30]: Probably re-word this a bit. Maybe just add a note saying that the image is not there yet.

Comment (hj-johannes-lee) [reply cmd/dlb_plugin/README.md]: I added!

**Review by mythi — APPROVED** [review]
+1

**Pull request merged**